server/neptune/workflows/activities/execute.go

```diff
@@ -41,3 +48,50 @@ func (t *executeCommandActivities) ExecuteCommand(ctx context.Context, request E
 		Output: string(out),
 	}, nil
 }
+
+type StringCommand struct {
+	Command        string
+	Dir            string
+	AdditionalEnvs map[string]string
+}
+
+func (l *StringCommand) Load() (string, error) {
+	cmd := exec.Command("sh", "-c", l.Command) // #nosec
+	cmd.Dir = l.Dir
+
+	finalEnvVars := os.Environ()
+	for key, val := range l.AdditionalEnvs {
+		finalEnvVars = append(finalEnvVars, fmt.Sprintf("%s=%s", key, val))
+	}
+
+	cmd.Env = finalEnvVars
+	out, err := cmd.CombinedOutput()
+
+	if err != nil {
+		return "", errors.Wrap(err, "executing cmd")
+	}
+
+	// Trim newline from res to support running `echo env_value` which has
+	// a newline. We don't recommend users run echo -n env_value to remove the
+	// newline because -n doesn't work in the sh shell which is what we use
+	// to run commands.
+	// n
+	return strings.TrimSuffix(string(out), "\n"), err
```

Comment:
Do we have a simple test for this?

Reply:
This was copied over from OSS including the comment but yeah this [test](https://github.com/lyft/atlantis/blob/fc64bc942af9a8311ae57c5d7e145ff981e10d98/server/neptune/workflows/activities/execute_test.go#L12-L13) fails if we remove that.